### ISSUE TITLE
I gives some classical composer's midi to it and val_loss cannot be converged

### ISSUE THREAD

**Issue by momo1986** [qlemaire22/Music-Generation#2]
My step:
1) Read all the notes of the converted classical music notes (e.g., bach and beethoven's midi record) .
2) Make it as npy file as the demo showed.
3) Train it with LSTM, set Dropout = 0.5, I see that the val_loss cannot be decreased after 5 epochs:

**467208/467208 [==============================] - 192s 410us/step - loss: 4.6073 - val_loss: 4.6297
Epoch 2/500
467208/467208 [==============================] - 185s 397us/step - loss: 4.5332 - val_loss: 4.6061
Epoch 3/500
467208/467208 [==============================] - 185s 396us/step - loss: 4.4899 - val_loss: 4.6165
Epoch 4/500
467208/467208 [==============================] - 186s 397us/step - loss: 4.4299 - val_loss: 4.6036
Epoch 5/500
467208/467208 [==============================] - 185s 395us/step - loss: 4.3399 - val_loss: 4.5027
Epoch 6/500
467208/467208 [==============================] - 186s 398us/step - loss: 4.2651 - val_loss: 4.5016
Epoch 7/500
467208/467208 [==============================] - 184s 394us/step - loss: 4.1843 - val_loss: 4.6069
Epoch 8/500
467208/467208 [==============================] - 185s 397us/step - loss: 4.0978 - val_loss: 4.5049
Epoch 9/500
467208/467208 [==============================] - 183s 392us/step - loss: 4.0121 - val_loss: 4.5427
Epoch 10/500
467208/467208 [==============================] - 187s 400us/step - loss: 3.9280 - val_loss: 4.5324
Epoch 11/500
467208/467208 [==============================] - 184s 395us/step - loss: 3.8455 - val_loss: 4.6753
Epoch 12/500
467208/467208 [==============================] - 186s 397us/step - loss: 3.7679 - val_loss: 4.5808
Epoch 13/500
467208/467208 [==============================] - 184s 394us/step - loss: 3.6956 - val_loss: 4.6103
Epoch 14/500
467208/467208 [==============================] - 186s 397us/step - loss: 3.6259 - val_loss: 4.6841
Epoch 15/500
467208/467208 [==============================] - 185s 396us/step - loss: 3.5617 - val_loss: 4.6676
Epoch 16/500
467208/467208 [==============================] - 185s 396us/step - loss: 3.5003 - val_loss: 4.6769
Epoch 17/500
467208/467208 [==============================] - 185s 396us/step - loss: 3.4441 - val_loss: 4.7305**

Is there any tips that I can adjust my networks?
About 250 classical midi songs,  is the trian dataset not big enough?
I get the data from this web-site?
http://www.piano-midi.de/
Looks like 250 songs is not enough for training, my question is：
1） Is there more classical midi for downloading?
2)  Is there any direction that I can change the network structure?
Thanks & Regards!
